src/main/java/mate/academy/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork(){
+        System.out.println("Excavator started its work.");
+    }
+    @Override
+    public void stopWork(){
+        System.out.println("Excavator stopped working.");
+    }
+}
```

Comment:
each file should end with exactly one empty line (it won't be displayed on GitHub). here you have 0, that's why this red symbol appears, fix everywhere
![image](https://github.com/mate-academy/jv-oop/assets/49767049/5c5f5636-cc60-48b0-952f-cc5370c5dad6)